### PR TITLE
Fixes modal reopen bug after edit

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile/tests/fixtures/transaction-completed-with-no-changes.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
[the remark this answers is not in the document]
Comment (tmitchellgcio) [reply src/applications/personalization/profile/tests/fixtures/transaction-completed-with-no-changes.json:1]: lol sorry about that. I will scrap that

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js [hunk 2]
```diff
@@ -143,4 +144,37 @@ describe('Modals on the personal information and content page', () => {
       sectionName: 'contact email address',
     });
   });
+
+  it('should clear transactions when successful transaction occurs', () => {
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:148]: I love everything about this PR but would suggest a couple very small tweaks:

1. Move this new test into its own `describe` block. I think it's fine for the test to live in this file, but this particular test isn't really testing the warning modals like all of the other tests.
2. And make the test description really clear that it's testing the ability to re-edit a piece of contact info after making a no-op save

**Review by erikphansen — CHANGES_REQUESTED** [review]
Looks great! Thanks so much for adding a test for this. I just had a couple requests to make things a little less confusing for our future selves.

**Review by erikphansen — APPROVED** [review]
🚀

**Pull request merged**